subgraphs/harvest-finance/tests/megaFactory.test.ts

```diff
@@ -0,0 +1,58 @@
+import { Address } from '@graphprotocol/graph-ts'
+import { assert, describe, test, afterEach, clearStore } from 'matchstick-as'
+import { helpers } from './helpers'
+import { RewardToken } from '../generated/schema'
+import { handleDeploymentCompleted } from '../src/megaFactory'
+import { createVault } from './vault.test'
+
+describe('MegaFactory', () => {
+  afterEach(() => {
+    clearStore()
+  })
+
+  test('Create vault, tokens and rewardTokens', () => {
```

Comment:
add describe that indicates the name of the handler we are testing here. same way we are doing in other tests